ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -500,23 +535,18 @@ const AdminUser = withUrlParams()(class extends React.Component<Props, State> {
                 containerStyle={styles.textInputContainer}
                 onChange={email => this.setContactEmail(email)}
             />
-            <TextInputWithLabel
-                labelText={'Free credits used'}
-                placeholder={updatedProfile.freeTierUsage}
-                inputId={'freeTierUsage'}
-                disabled={true}
-                inputStyle={{width: '6.5rem', ...styles.backgroundColorDark}}
-                containerStyle={styles.textInputContainer}
+            <FreeCreditsUsage
+              aboveLimit={this.usageIsAboveLimit()}
```

Comment:
nit: I'd name this `isAboveLimit`. Generally booleans starting with forms of "be" are more recognizable IMO.